### PR TITLE
Added wait time on second read counter operation

### PR DESCRIPTION
**Pull request by daniele-moro**

### CODE REVIEW

File: tests/ptf/fabric_test.py [hunk 1]
```diff
@@ -1172,6 +1173,8 @@ def runUpstreamV4Test(self, pkt, tagged2, mpls, line_enabled=True):
             tagged2=tagged2, mpls=mpls, verify_pkt=line_enabled)
 
         # Verify that upstream counters were updated as expected.
+        if self.is_bmv2():
```
Comment (ccascone) [inline tests/ptf/fabric_test.py]:
```suggestion
        if not self.is_bmv2():
```

Comment (ccascone) [reply tests/ptf/fabric_test.py]: Applies to other places where the sleep is invoked

File: tests/ptf/base_test.py [hunk 2]
```diff
@@ -639,7 +646,7 @@ def send_request_add_entry_to_group(self, t_name, mk, grp_id):
         self.push_update_add_entry_to_group(req, t_name, mk, grp_id)
         return req, self.write_request(req, store=(mk is not None))
 
-    def read_counter(self, c_name, c_index):
+    def read_counter(self, c_name, c_index, wait_time=1):
```
Comment (ccascone) [inline tests/ptf/base_test.py]: wait_time is unused

**Comment by ccascone** [comment]
Travis complains about checkstyles. Can you please fix that?

**Comment by daniele-moro** [comment]
> Travis complains about checkstyles. Can you please fix that?

**Comment by daniele-moro** [comment]
> Travis complains about checkstyles. Can you please fix that?

Checkstyle fixed!

**Pull request merged**